F suggestions ts (#7)
krlmlr/tidyprog: script/72-forward.Rmd

```
</details>

So does everyone who wants to create any functions in the framework of the tidyverse need deep knowledge about tidy evaluation?

> The answer is, it depends: often enough, you can avoid the need for this knowledge.

Reviewer: ```suggestion
The answer is, it depends: often enough, things "just work".
```
